src/components/marketoForms/MarketoForm.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from "react";
+import { MarketoFormStyleOverwrite, SuccessText } from "./styles";
+
+declare global {
+  interface Window {
+    MktoForms2: {
```

Comment:
This could be marked optional so you make sure to check for undefined before using it

```suggestion
    MktoForms2?: {
```